pkg/credentials/https/https_secret.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2021 kubeflow.org.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package https
+
+import (
+	"github.com/kubeflow/kfserving/pkg/constants"
+	v1 "k8s.io/api/core/v1"
+	"strings"
+)
+
+// Create constants -- baseURI
+const (
+	HTTPSHostURI = "https-base-uri"
+	HEADER       = "header"
```

Comment:
why is key "header" here but "headers" in the readme?

Reply:
Good catch! Thanks